db/migrate/20170725120715_add_gender_to_user.rb

```diff
@@ -0,0 +1,5 @@
+class AddGenderToUser < ActiveRecord::Migration[5.0]
+  def change
+    add_column :users, :gender, :string
```

Comment:
http://edgeapi.rubyonrails.org/classes/ActiveRecord/Enum.html

Look at type of field for enums there.